debian/control

```diff
@@ -0,0 +1,15 @@
+Source: python3-avocado-framework
+Section: python
+Priority: optional
+Maintainer: David Negreira <david.negreira@canonical.com>
+Build-Depends: debhelper (>=11~), dh-python, python3-all, python3-setuptools, python3-pip, python3-dev, python3-sphinx, python3-sphinx-rtd-theme
+Standards-Version: 4.6.2
+
+Package: python3-avocado-framework
+Architecture: all
+Homepage: https://github.com/avocado-framework/avocado
+X-Python3-Version: >=3.7
```

Comment:
Is this correct for 103? Or >=3.6 would work as well?

Reply:
For 103.0 we advertise (on `setup.py`):

```python
            "Programming Language :: Python :: 3.7",
            "Programming Language :: Python :: 3.8",
            "Programming Language :: Python :: 3.9",
            "Programming Language :: Python :: 3.10",
            "Programming Language :: Python :: 3.11",
            "Programming Language :: Python :: 3.12",
```

And test (`.github/workflows/ci.yml`):

```yaml
    strategy:
      matrix:
        python-version: [3.7, 3.8, 3.9, 3.10.0, 3.11, 3.12.0]
```